### PR TITLE
Not found response

### PR DESCRIPTION
**Pull request by grace000**
This PR supports this [story.](https://trello.com/c/rccluDB4/5-returning-a-not-found-response-725)

Client receives a 200 response for a simple get request. Any other request will create a not found response. I'm in the process of breaking up the request handler class. My next approach will be to assign those tasks to the server instead, but I welcome any design recommendations!

I'm also aware of an interesting issue that may be stemming from the input stream reader library that I'm working with. After several successful responses, requests are empty and the server fails. Please let me know if you have any advice for a input stream reading. 📖

### CODE REVIEW

**Comment by BWoodfork** [comment]
There's a bunch of `.idea` files that got committed. Should those have been committed or should they be gitignored?

File: src/main/java/github/grace000/erversay/response/ResponseBuilder.java [hunk 1]
```diff
@@ -1,17 +1,18 @@
 package github.grace000.erversay.response;
 
 public class ResponseBuilder {
-    private int code = 200;
+    private String code = "200";
     public String protocol = "HTTP/1.1";
     private String status = "OK";
 
-    public ResponseBuilder withCode(int code){
+//    public ResponseBuilder withCode(int code){
```
Comment (BWoodfork) [inline src/main/java/github/grace000/erversay/response/ResponseBuilder.java]: Dead code.

File: src/main/java/github/grace000/erversay/request/RequestRouter.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package github.grace000.erversay.request;
+
+import java.util.Hashtable;
+import static github.grace000.erversay.Constants.*;
+
+public class RequestRouter {
+    private Hashtable<String, String> routeTable = new Hashtable<>();
```
Comment (BWoodfork) [inline src/main/java/github/grace000/erversay/request/RequestRouter.java]: Any reason you used `Hashtable` instead of `Hashmap`?

Comment (grace000) [reply src/main/java/github/grace000/erversay/request/RequestRouter.java]: After further research I realized that I shouldn't be using Hashtable. I didn't realize that they both provided key value pairs. Thanks!

**Pull request merged**